sentry-samples/sentry-samples-log4j2/src/main/java/io/sentry/samples/log4j2/Main.java

```diff
@@ -0,0 +1,26 @@
+package io.sentry.samples.log4j2;
+
+import java.util.UUID;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.ThreadContext;
+
+public class Main {
+  private static final Logger LOGGER = LogManager.getLogger(Main.class);
+
+  public static void main(String[] args) {
+    LOGGER.debug("Hello Sentry!");
```

Comment:
```suggestion
    // The SDK was initialized through the appender configuration because a DSN was set there.
    // Update the DSN in log4j2.xml to see these events in your Sentry dashboard.
    LOGGER.debug("Hello Sentry!");
```